### PR TITLE
newlog: fix file naming for logs

### PR DESCRIPTION
**Pull request by europaul**
Revert the file naming inside devUpload and keepSentQueue directories to the original format `dev.log.<timestamp>.gz` to keep the compatibility with previous versions of the newlogd.

This is necessary e.g. in cases when EVE goes through an unsuccessful update and has to fallback to a version before the new naming convention. This way the logic inside newlogd, loguploader and edge-view can still understand the log files inside `/persist/newlog` and not throw an error.

### CODE REVIEW

File: docs/LOGGING.md [hunk 1]
```diff
@@ -115,7 +116,7 @@ The uploading is controlled on a scheduled timer. When the timer fires, the "log
 
 The "loguploader" collects stats of round-trip delay, controller CPU load percentage and log batch processing time. The current EVE implementation does not use those stats in calculating the uploading timer values.
 
-The already uploaded gzip files with app logs are moved to /persist/newlog/keepSentQueue directory or removed in case of dev.log.upload files.
+The already uploaded gzip files with app logs are moved to /persist/newlog/keepSentQueue directory or removed in case of dev.log files.
```
Comment (eriknordmark) [inline docs/LOGGING.md:119]: All 'dev.log*' named files can not be removed. can they?

Comment (naiming-zededa) [reply docs/LOGGING.md:119]: I think they have a copy in the 'keep' directory already in the latest newlog change. so they should be removed.

Comment (europaul) [reply docs/LOGGING.md:119]: @naiming-zededa is right, but I understand that the sentence is confusing - I'll rewrite

**Review by eriknordmark — APPROVED** [review]
Run tests

**Review by OhmSpectator — APPROVED** [review]
Good that it fixes the current problem. Let's merge it soon.

But could we add some logic on top to avoid fatals with log formats in the future? For example, not ending up with a log.Fatal because of invalid log names, or maybe some cleanup logic to remove problematic files on request? I’m not sure where exactly this would fit, but just an idea...

**Comment by eriknordmark** [comment]
> But could we add some logic on top to avoid fatals with log formats in the future? For example, not ending up with a log.Fatal because of invalid log names, or maybe some cleanup logic to remove problematic files on request? I’m not sure where exactly this would fit, but just an idea...

Yes, but please lets do that as a separate PR.
Currently tests are failing because some tests run master or a PR for a while and then they go back to 13.4.X and fatal due to this issue. So I want master to be usable again first.

**Pull request merged**